### ISSUE TITLE
Relative paths showing up in generated JSON file

### ISSUE THREAD

**Issue by creativedrewy** [solana-mobile/app-publishing-spec#125]

**Comment by creativedrewy** [comment]
From Tulip

```
"media":[{"purpose":"icon","uri":"../android/app/src/main/res/mipmap-mdpi/ic_launcher.png","mime":"image/png","size":2765,"sha256":"b57fa5b1e9d73135588a15b8d82ebbc299c95c836710aed036ec8111700f6914","width":48,"height":48},{"purpose":"screenshot","uri":"../android/app/src/main/assets/screenshots/lending.png","mime":"image/png","size":1087154,"sha256":"967cacaf62295fc8245afa46dd83445c706a6175f755d89bc0c24463d48f8bce","width":1304,"height":2624}],"files":[{"purpose":"install","uri":"../build/outputs/apk/release/myapp-release.apk","mime":"application/vnd.android.package-archive","size":26041583,"sha256":"5d76a603e7a0a9c88f4ac64f79f5292cbca5f0af8d990a5b00b34ccae948b2a5"}]
```

**Comment by creativedrewy** [comment]
Fixed by #126
